docs/user_guide/building_blocks/config_objects.md

```diff
@@ -53,9 +125,374 @@ given via the ```--variables``` argument, accessing information from columns bas
 the `expression` of the Variable and storing them in histograms with binning defined
 via the `binning` argument of the Variable.
 
-The list of possible keyword arguments can be found in
-Variable. The values in the ```expression``` argument can
+The list of possible keyword arguments can be found in the order documentation for the class
+{external+order:py:class}`order.variable.Variable`. The values in the ```expression``` argument can
 be either a one-dimensional or a more dimensional array. In this second case the information is
 flattened before plotting. It is to be mentioned that
 {py:attr}`~columnflow.columnar_util.EMPTY_FLOAT` is a columnflow internal null value and
 corresponds to the value ```-9999.0```.
+
+### Category
+
+Categories built to investigate specific parts of the phase-space, for example for plotting.
+These objects are described in [the Channel and Category](https://python-order.readthedocs.io/en/latest/quickstart.html#channel-and-category)
+part of the Quickstart section of the order documentation. A specificity of columnflow regarding
+this class is the fact that the `selection` argument of this object is expected to take the name of
+an object of the {py:class}`~columnflow.categorization.Categorizer` class instead of a boolean
+expression in a string format.
+Adding a category to the Config object is possible through the
+{py:func}`~columnflow.config_util.add_category()` method. An example for an inclusive category with
+the Categorizer `cat_incl` defined in the cms_minimal analysis template is given below:
+
+```python
+add_category(
+    cfg,
+    id=1,
+    name="incl",
+    selection="cat_incl",
+    label="inclusive",
+)
+```
+
+It is recommended to always add an inclusive category with id=1 or name="incl" which is used
+in various places, e.g. for the inclusive cutflow plots and the "empty" selector.
```

Comment:
because we wanted to add it to the standard cf functions if I recall correctly and an empty selector would probably be much less interesting without an inclusive category. But I can just delete that